CONTRIBUTING.md

```diff
@@ -10,4 +10,10 @@
 1. สั่ง Push เข้าไปที่ Branch (`git push origin feature/fooBar`)
 1. เปิด Pull Request
 
-มาร่วมสร้างหนังสือ Data Engineering สำหรับชาวไทยทุกคนกันนะครับ 😉
\ No newline at end of file
+## ขั้นตอนการตรวจไฟล์ก่อนจะ commit (pre-commit)
+
+1. Install [pre-commit]('https://pre-commit.com/)
+1. Run `pre-commit install` on this Repo
+1. Run `pre-commit run --all-files`
```

Comment:
if you add pre-commit to existing repo, you need to force run pre-commit on all files via `pre-commit run --files **/*`

But, if for normal workflow, just creating a commit via `git commit -m "$MESSAGE" would automatically trigger pre-commit hooks. This is because by default, pre-commit only run on files you added to a current commit.

Reply:
So we don't have to run `pre-commit run --files **/*`  because it run when `commit` right?

Reply:
✅